FAQ/en.md

```diff
@@ -0,0 +1,61 @@
+# Frequently Asked Questions
+
+## General
+
```

Comment:
More questions to add:

1. What is Quaver?
2. How is Quaver different from other games?
3. What platforms will Quaver be released on?

Reply:
Added all 3.